packages/web-components/src/components/va-omb-info/va-omb-info.tsx

```diff
@@ -54,6 +54,11 @@ export class VaOmbInfo {
    */
   @Prop() resBurden?: number;
 
+/**
+ * If `true`, clicking outside the modal will close it.
+ */
+  @Prop() modalClickToClose?: boolean = true;
```

Comment:
I chose to default this to true because of personal preference, but if we want to default it to false to maintain current functionality we can do that.

Reply:
I think we should be consistent across components. The default for modal is false so I will update this to be false too.